ank/doc/swdesign/README.md

```diff
@@ -529,6 +529,223 @@ Needs:
 - impl
 - utest
 
+### `ank apply [-d] [--agent agent_name] <manifest.yaml> ...`
+
+#### Ankaios manifest
+
+The Ankaios manifest is a YAML (or a JSON) file composed of a list of workload specifications under the `workloads` keyword.
+
+```yaml
+# Example of a list of two workload specifications with the names 'nginx' and 'hello1'.
+workloads:
```

Comment:
Is this format the same as the "startConfig" ? Or does it share a part of the CompleteState? We have made changes in both. Does it mean, that this format shall be changed (has changed too)?

* Double-check please, if this format is still up-to-date after merging changes from the current main.
* Think about making here rather a reference to the format of the startConfig if the format is shared with something.

Reply:
ok, I will check!